qa/vagrant/src/main/java/org/elasticsearch/packaging/util/Archives.java

```diff
@@ -0,0 +1,239 @@
+                    "elasticsearch");
+            }
+        }
+        sh.runAndAssertSuccess("chown", "-R", "elasticsearch:elasticsearch", installPath.toString());
+    }
+
+    public static void verifyArchiveInstallation(Installation installation, Distribution distribution) {
+        // on Windows for now we leave the installation owned by the vagrant user that the tests run as. Since the vagrant account
+        // is a local administrator, the files really end up being owned by the local administrators group. In the future we'll
+        // install and run elasticesearch with a role user on Windows
+        final String owner = Platforms.WINDOWS
+            ? "BUILTIN\\Administrators"
+            : "elasticsearch";
+
+        verifyOssInstallation(installation, distribution, owner);
+        if (distribution.flavor == Distribution.Flavor.DEFAULT) {
+            verifyDefaultInstallation(installation, distribution, owner);
+        }
+    }
+
+    private static void verifyOssInstallation(Installation es, Distribution distribution, String owner) {
+        Stream.of(
+            es.home,
+            es.config,
+            es.plugins,
+            es.modules,
+            es.logs
+        ).forEach(dir -> assertThat(dir, file(Directory, owner, owner, p755)));
+
+        assertThat(Files.exists(es.data), is(false));
+        assertThat(Files.exists(es.scripts), is(false));
+
+        assertThat(es.home.resolve("bin"), file(Directory, owner, owner, p755));
+        assertThat(es.home.resolve("lib"), file(Directory, owner, owner, p755));
+        assertThat(Files.exists(es.config.resolve("elasticsearch.keystore")), is(false));
+
+        Stream.of(
+            "bin/elasticsearch",
+            "bin/elasticsearch-env",
+            "bin/elasticsearch-keystore",
+            "bin/elasticsearch-plugin",
+            "bin/elasticsearch-translog"
+        ).forEach(executable -> {
+
+            assertThat(es.home.resolve(executable), file(File, owner, owner, p755));
+
+            if (distribution.packaging == Distribution.Packaging.ZIP) {
+                assertThat(es.home.resolve(executable + ".bat"), file(File, owner));
+            }
+        });
+
+        if (distribution.packaging == Distribution.Packaging.ZIP) {
+            Stream.of(
+                "bin/elasticsearch-service.bat",
+                "bin/elasticsearch-service-mgr.exe",
+                "bin/elasticsearch-service-x64.exe"
+            ).forEach(executable -> assertThat(es.home.resolve(executable), file(File, owner)));
+        }
+
+        Stream.of(
+            "elasticsearch.yml",
+            "jvm.options",
+            "log4j2.properties"
+        ).forEach(config -> assertThat(es.config.resolve(config), file(File, owner, owner, p660)));
+
+        Stream.of(
+            "NOTICE.txt",
+            "LICENSE.txt",
+            "README.textile"
+        ).forEach(doc -> assertThat(es.home.resolve(doc), file(File, owner, owner, p644)));
```

Comment:
This isn't passing for me:
```
Expected: "file/directory: "<File> with owner "elasticsearch" with group "elasticsearch" with posix permissions [<OWNER_READ>,<OWNER_WRITE>,<GROUP_READ>,<OTHERS_READ>]
     but: path </tmp/elasticsearch/NOTICE.txt>Has permissions [GROUP_WRITE, OWNER_READ, GROUP_READ, OWNER_WRITE, OTHERS_READ]
```

Reply:
Weird, it passes locally for me and in CI. Which box and test?

Reply:
Actually if you got a full reproduce line that would be best - I'm pretty sure the randomization doesn't affect anything about the current version installs, but best to check anyway

Reply:
I ran:
```
./gradlew -p qa/vagrant/ vagrantCentos7#javaPackagingTest
```

I'm pulling and trying again.

Reply:
Yeah. That command fails for me.

Reply:
Weird, still not seeing it. If you still have it mind sharing the test seed it used

Reply:
I got this:
```
REPRODUCE WITH: ./gradlew :qa:vagrant:vagrantCentos7#javaPackagingTest -Dtests.seed=924BDF23E0249263 
```

Though I'm not sure how useful that is.